koku/api/report/queries.py

```diff
@@ -77,7 +74,11 @@ class ProviderMap(object):
                 },
                 'report_type': {
                     'costs': {
+                        'aggregate': {'value': Sum('unblended_cost'),
+                                      'cost': Sum('unblended_cost')},
```

Comment:
I guess it depends on which tradeoffs we want to make - Having a consistent set of outputs or eliminating redundancies.  

Right now, I erred on the side of not breaking the existing API and doing just adding in the new feature. Do you want this to be an API-breaking change or would you like to preserve compatibility until we can coordinate a future change with the UI to remove the other value?

Reply:
Was it just `value` previously? Why not just keep value there and we're good. The changes here are just to add cost into the instance and storage APIs.

Reply:
Again, it's a question of consistency. All reports should supply the same set of data labels. We shouldn't have cost reports returning data named `cost` but instance-type and storage reports returning the same data, but with a different name of `value` or `total`.

Reply:
I agree, and this isn't big enough to hold things up. I wasn't saying keep `cost` only here, but rather `value`. Prior to this PR all 3 endpoints returned `value`. I'm not saying I love that word, and it means cost for cost APIs, and usage for inventory APIs. 

My personal preference would be to mirror what we did for OCP where we have usage, and charge returned instead of a more generic term. But as you said--breaking API change.